draft-irtf-cfrg-hpke.md

```diff
@@ -4628,3 +4775,57 @@ L: 32
 exported_value:
 84f3466bd5a03bde6444324e63d7560e7ac790da4e5bbab01e7c4d575728c34a
 ~~~
+
+# Acknowledgements
+
+The authors would like to thank
+Joël Alwen,
+Jean-Philippe Aumasson,
+David Benjamin,
+Benjamin Beurdouche,
+Bruno Blanchet,
+Frank Denis,
+Stephen Farrell,
+Scott Fluhrer,
+Eduard Hauck,
+Scott Hollenbeck,
+Kevin Jacobs,
+Burt Kaliski,
+Eike Kiltz,
+Julia Len,
+John Mattsson,
+Christopher Patton,
+Doreen Riepel,
+Raphael Robert,
+Michael Rosenberg,
+Michael Scott,
+Martin Thomson,
+Steven Valdez,
+Riad Wahby,
+and other contributors in the CFRG for helpful feedback that greatly improved this document.
+
+{::comment}
+[rfced] Throughout the text, the following terminology appears to
+be used inconsistently. Please review the usage and let us know if/how
+they may be made consistent.
+
+secret export vs. Secret Export
+  (e.g., secret export interface vs. Secret Export interface)
+
+context vs. Context
+  (e.g., Context structure vs. context string)
+{:/comment}
+
+{::comment}
+[rfced] Please review the "Inclusive Language" portion of the online
+Style Guide <https://www.rfc-editor.org/styleguide/part2/#inclusive_language>
+and let us know if any changes are needed.
+
+In addition, please consider whether "traditional" should be updated for clarity.
+While the NIST website
+<https://www.nist.gov/nist-research-library/nist-technical-series-publications-author-instructions#table1>
+indicates that this term is potentially biased, it is also ambiguous.
+"Tradition" is a subjective term, as it is not the same for everyone.
```

Comment:
"Traditional" here is not in terms of a human tradition (say, Thanksgiving).  Rather, it describes what is in the literature.